### ISSUE TITLE
On the cookie banner, change "This site uses cookies" to something that would also cover localstorage and sessionstorage

### ISSUE THREAD

**Issue by anth-volk** [PolicyEngine/policyengine-app#1596]
Just to ensure full compliance, this should be broadened to cover localstorage and sessionstorage. We'll need to discuss the wording in this thread before opening a PR.

**Comment by daemon-reconfig** [comment]
<img width="786" alt="Screenshot 2024-04-13 at 11 04 46 AM" src="https://github.com/PolicyEngine/policyengine-app/assets/69844188/d7f4faa8-6752-487c-b90a-bddf2ecd8c4b">

should we add it here?

**Comment by anth-volk** [comment]
Yes, it should be added to the existing cookies banner, but first we need to think up what to write. Any ideas?

**Comment by daemon-reconfig** [comment]
this site uses cookies which also includes your session and local storage to improve your experience. something of sorts i guess

**Comment by anth-volk** [comment]
Technically, cookies, sessionStorage, and localStorage are all different, so I'd be hesitant to use that phrasing. Perhaps something more like:

This site utilizes your browsing data to improve your experience.

But I'm hoping to hear more broadly from members of the team, as we want to make sure this doesn't sound like we're harvesting and/or selling data, as we definitely are not.

**Comment by dotslashbit** [comment]
@anth-volk  How about also including a link to the data usage page where users can read and understand how there data is being used and what data is being collected

**Comment by anth-volk** [comment]
@dotslashbit I'm not sure we have a particular data usage page _per se_, but we could at least link to the terms & conditions page.

**Comment by MaxGhenis** [comment]
Thank you for filing this issue. Closing as part of the `policyengine-app` (v1) wind-down: policyengine.org is now served by [`policyengine-app-v2`](https://github.com/PolicyEngine/policyengine-app-v2), and this repository is being archived, so this issue is frozen against a codebase that no longer deploys anywhere.

If this is still relevant on the current site, please re-file it against [`policyengine-app-v2`](https://github.com/PolicyEngine/policyengine-app-v2) — we'd welcome it there. Apologies that this sat open through the migration.
